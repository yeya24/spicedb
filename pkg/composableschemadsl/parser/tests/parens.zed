definition foo {
    permission bar = (maz & beh)
}