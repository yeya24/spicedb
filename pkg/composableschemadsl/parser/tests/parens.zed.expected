NodeTypeFile
  end-rune = 50
  input-source = parens test
  start-rune = 0
  child-node =>
    NodeTypeDefinition
      definition-name = foo
      end-rune = 50
      input-source = parens test
      start-rune = 0
      child-node =>
        NodeTypePermission
          end-rune = 48
          input-source = parens test
          relation-name = bar
          start-rune = 21
          compute-expression =>
            NodeTypeIntersectExpression
              end-rune = 47
              input-source = parens test
              start-rune = 39
              left-expr =>
                NodeTypeIdentifier
                  end-rune = 41
                  identifier-value = maz
                  input-source = parens test
                  start-rune = 39
              right-expr =>
                NodeTypeIdentifier
                  end-rune = 47
                  identifier-value = beh
                  input-source = parens test
                  start-rune = 45